sortfilterproxymodel.qbs

```diff
@@ -31,6 +31,7 @@ Product
     }
 
     cpp.includePaths: [ "." ]
+    cpp.cppFlags: "-std=c++11"
```

Comment:
@johnbivens I have seen that answer but wanted to doublecheck. Already added a hooray smilie above =)